addons/mail/static/src/components/mobile_messaging_navbar/mobile_messaging_navbar.js

```diff
@@ -6,6 +6,16 @@ const { Component } = owl;
 
 export class MobileMessagingNavbar extends Component {
 
+    /**
+     * @returns {mail.mobile_messaging_navbar}
+     */
+    get mobileMessagingNavbar() {
+        return (
+            this.messaging &&
+            this.messaging.models['mail.mobile_messaging_navbar'].get(this.props.mobileMessagingNavbarLocalId)
+        );
```

Comment:
4 lines is a bit too much for this idea, we shouldn't really care about the length of a single here I think.